### PR TITLE
Added support for nullable and warning wave error code groups

### LINKED ISSUE #4: I think this tool will miss doc links to articles with multiple errors
First off, thanks for writing this. It's super useful! 

Over time, I'm working on different ways to minimize the number of C# compiler messages not documented. See dotnet/docs#24002.

As I work on those areas, I'm starting to create single articles that provide guidance for a related set of errors / warnings.  You can see two examples here:

- [Nullable warnings](https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-messages/nullable-warnings)
- [Warning waves](https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-messages/warning-waves)

It appears that `GetDocLink` will miss any errors and warnings in those two articles (and others I'll be writing going forward).

I can submit a PR if you want.

### PR DESCRIPTION
**Pull request by peteraritchie**
and expanded markdown escape.

Fixes #4

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -154,6 +154,20 @@ private static async Task<IReadOnlyList<EnumMemberDeclarationSyntax>> GetErrorCo
                 codes.Add(code, href);
             }
 
+            foreach (var item in root!.Items.Where(e => e.Items == null && !string.IsNullOrWhiteSpace(e.DisplayName)))
+            {
+                var displayName = item.DisplayName.Replace(", ", ",");
+                var subCodes = displayName.Split(',');
```
Comment (thomaslevesque) [inline Program.cs]: Nit: you could use `Regex.Split(item.DisplayName, @"\s*,\s*")` to account for the optional whitespace.

**Review by thomaslevesque — APPROVED** [review]
Thanks @peteraritchie! I just have one small, non-blocking comment, feel free to take it into account or not...

**Review by BillWagner — APPROVED** [review]
This all LGTM @peteraritchie 

Thanks for submitting a fix so quickly.

File: Program.cs (hunk 1 above)
Comment (peteraritchie) [reply Program.cs]: I like it, thanks for the suggestion!

**Comment by thomaslevesque** [comment]
Thanks again, @peteraritchie!

**Pull request merged**